### DISCUSSION TITLE
Filter East/West Coast Worlds on World Hopper

### DISCUSSION THREAD

**Discussion by A-q-p-W** [runelite/runelite#17559] [Ideas]
I am in EU and ping to East Coast worlds is 90-100ms while West Coast can be 200ms+. Would be great to be able to hop through East Coast worlds with World Hopper keyboard shortcuts without going to West Coast ones.

**Comment by Alexgopen** [comment]
Seconding this, I currently filter to USA region, but I want to filter to only US West.  Splitting up the USA region filter to US East and US West would help.

**Comment by setter43** [comment]
Also:
A) Option to turn region filter off.
B) Option to select multiple regions.

Perhaps this could be achieved with tick boxes. Toggle all of or toggle multiple.
